samples/grpc-server-aot/build.gradle

```diff
@@ -0,0 +1,67 @@
+plugins {
+    id 'java'
+    id 'org.springframework.boot' version '3.4.3'
+    id 'io.spring.dependency-management' version '1.1.6'
+    id 'org.springframework.boot.aot' version '3.4.3'
```

Comment:
We should use the `id 'org.graalvm.buildtools.native' version '0.10.5'` plugin instead which covers both AOT and Native use.